library/Icinga/Web/View/Helper/IcingaCheckbox.php

```diff
@@ -11,7 +11,8 @@ public function icingaCheckbox($name, $value = null, $attribs = null, array $che
             $attribs['id'] = $this->view->protectId('icingaCheckbox_' . $name);
         }
 
-        $attribs['class'] = (isset($attribs['class']) ? $attribs['class'] . ' ' : '') . 'sr-only';
+        $attribs['class'] = (isset($attribs['class']) ? $attribs['class'] . ' ' : '');
```

Comment:
The `. ' '` only exists due the `sr-only` class. Please remove that as well.

Reply:
You're right. I removed it.